src/main/java/utils/filereaders/CsvFileReader.java

```diff
@@ -0,0 +1,29 @@
+package utils.filereaders;
+
+import utils.CustomFileReader;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CsvFileReader implements CustomFileReader {
+
+    @Override
+    public List<String> readFile(File file) {
```

Comment:
go through all the created classes and properly format the code, can use ctrl alt l in intellij, i think that is the command, not sure